CHANGELOG.md

```diff
@@ -6,7 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 The changes documented here do not include those from the original repository.
 
-## [Unreleased]
+## [Version 2.3.1]
+
+### Fixes
+- Fixes notification click when notification is delievered while the app is in foreground (https://outsystemsrd.atlassian.net/browse/RMET-3559).
+- Removes protected/reserverd keys from being sent as deeplink parameters data when clicking notification, and also when receiving in-app notifications. (https://outsystemsrd.atlassian.net/browse/RMET-3559).
+
+### Chores:
+- Update `OSFirebaseMessagingLib` Android library to version `1.2.1`.
```

Comment:
does it make sense to log these lib upgrades? since the `Fixes` basically explains what the new version does

Reply:
yeah maybe it's not necessary, I'll remove it